sklearn/linear_model/ridge.py

```diff
@@ -867,10 +871,12 @@ def fit(self, X, y, sample_weight=None):
             #fit_params = {'sample_weight' : sample_weight}
             fit_params = {}
             gs = GridSearchCV(Ridge(fit_intercept=self.fit_intercept),
-                              parameters, fit_params=fit_params, cv=self.cv)
+                              parameters, fit_params=fit_params, cv=self.cv,
+                              scoring = self.scoring)
```

Comment:
Why should it need to fail on master branch? (So I need to change the test in master branch?)
And I'm still curious about why the test should compare `alpha_`of RidgeCV with a specific cv with `alpha_` of GridSearchCV (and expect they are not equal?). 
The reason I'm curious is that RidgeCV with specified `cv` actually uses GridSearchCV to find `alpha_`, so if I use the GridSearchCV (with the same settings, i.e. `cv`, `alpha` and `scoring`), it should provide the same result.